data-prepper-plugins/http-source/src/main/java/com/amazon/dataprepper/plugins/source/loghttp/HTTPSourceConfig.java

```diff
@@ -32,15 +29,31 @@
     static final int DEFAULT_MAX_CONNECTION_COUNT = 500;
     static final int DEFAULT_MAX_PENDING_REQUESTS = 1024;
 
+    @JsonProperty("port")
     private int port = DEFAULT_PORT;
+
     @JsonProperty("request_timeout")
     private int requestTimeoutInMillis = DEFAULT_REQUEST_TIMEOUT_MS;
+
+    @JsonProperty("thread_count")
     private int threadCount = DEFAULT_THREAD_COUNT;
+
+    @JsonProperty("max_connection_count")
     private int maxConnectionCount = DEFAULT_MAX_CONNECTION_COUNT;
+
+    @JsonProperty("max_pending_requests")
     private int maxPendingRequests = DEFAULT_MAX_PENDING_REQUESTS;
+
+    @JsonProperty(SSL)
     private boolean ssl;
+
+    @JsonProperty(SSL_CERTIFICATE_FILE)
     private String sslCertificateFile;
+
+    @JsonProperty(SSL_KEY_FILE)
```

Comment:
Is there a difference between upper case and lower case JsonProperties? I am wondering why `ssl_key_password` and `SSL_KEY_FILE` are different.

Reply:
`SSL_KEY_FILE` is a static final field whereas `"ssl_key_password"` is a string literal. The former is used in some error strings so I kept it as constant. We could make them all constants if we wanted.

Reply:
oh, I missed the quotes. For a second I thought the properties were defined in a different format. I'd prefer consistency but will defer to your preference here. I don't want to hold this up.